atlassian-terraform/terraform.tfvars

```diff
@@ -0,0 +1,93 @@
+# Sumo Logic - Atlassian Terraform
+# Configure Sumo Logic, Jira Cloud, Jira Server (Onprem), OpsGenie, BitBucket credentials and installation options.
+
+# Sumologic
+sumo_access_id = "" # https://help.sumologic.com/Manage/Security/Access-Keys
```

Comment:
To continue my above message: 
`sumo_access_id =  "<YOUR SUMO ACCESS ID>" # Please replace <YOUR SUMO ACCESS ID> (including brackets) with your Sumo Access ID`

Reply:
Added